sessionctx/variable/session.go

```diff
@@ -737,15 +737,16 @@ func NewSessionVars() *SessionVars {
 	}
 	vars.KVVars = kv.NewVariables(&vars.Killed)
 	vars.Concurrency = Concurrency{
-		IndexLookupConcurrency:     DefIndexLookupConcurrency,
```

Comment:
We can set `DefIndexLookupConcurrency` as `ConcurrencyUnset`

Reply:
Updated. PTAL